AQ3DC/Classes/Measure.py

```diff
@@ -478,109 +489,109 @@ def __SignMeaningDentalAngle(self):
             self.line2[2]["name"],
         ]
         # print("function angle")
-        if check(lst_measurement, upper_right_back):
-            self.lr_sign_meaning = "D"
-            self.ap_sign_meaning = "L"
-            self.si_sign_meaning = "DR"
+        if check(lst_measurement, UPPER_RIGHT_BACK):
+            self.ap_sign_meaning = "D"
+            self.si_sign_meaning = "L"
+            self.lr_sign_meaning = "DR"
             if self.lr > 0:
-                self.lr_sign_meaning = "M"
+                self.ap_sign_meaning = "M"
 
             if self.ap > 0:
-                self.ap_sign_meaning = "B"
+                self.si_sign_meaning = "B"
 
             if self.si > 0:
-                self.si_sign_meaning = "MR"
+                self.lr_sign_meaning = "MR"
```

Comment:
Your intuition is good, what I did is wrong. I must change the condition in the if to feet with inside of the if.

Also good remark, it would be good to have some explanation why is like this. I have a powerpoint with all the condition, I will put this in docs folder.